src/vaults/VaultEventsFetcher.ts

```
@@ -48,9 +47,15 @@ export default class VaultEventsFetcher implements EventsFetcherInterface {
     }
 
     // now we first order events
-    const allEvents: TransactionEvents[] = _sortBy(
-      [...depositEvents, ...withdrawEvents, ...transferEvents],
-      (ev: TransactionEvents) => [ev.event.blockNumber, ev.event.transactionIndex, ev.event.logIndex]
+    const allEvents: TransactionEvents[] = [...depositEvents, ...withdrawEvents, ...transferEvents].sort(
+      (event1, event2) => {
+        if (event1.event.blockNumber !== event2.event.blockNumber)
+          return event1.event.blockNumber - event2.event.blockNumber;
+        if (event1.event.transactionIndex !== event2.event.transactionIndex)
+          return event1.event.transactionIndex - event2.event.transactionIndex;
+        if (event1.event.logIndex !== event2.event.logIndex) return event1.event.logIndex - event2.event.logIndex;
+        return 0;
```

Last case should not be possible no? Can we throw an error instead of 0?